src/website/shared/logs/logs.py

```diff
@@ -186,4 +186,42 @@ def get_dict(
             else:
                 grouped_activity_log[suggestion_id] = [event]
 
-        return grouped_activity_log
+        # Second pass to fold repeated package actions by user
+        folded_activity_log = {}
+        for suggestion_id, events in grouped_activity_log.items():
+            suggestion_log = []
+
+            accumulator = None
+            for event in events:
+                if not event["action"].startswith("derivations."):
```

Comment:
```suggestion
                if not event["action"].startswith("derivations"):
```

That period kinda rubs me the wrong way :rofl: